src/bp-moderation/bp-moderation-filters.php

```diff
@@ -654,3 +654,84 @@ function bb_moderation_admin_repair_old_moderation_data() {
 		);
 	}
 }
+
+
+/**
+ * Function which get next recepients list for block member in message section.
+ */
+function bp_nouveau_next_recepient_list_for_blocks() {
+	$post_data = filter_input( INPUT_POST, 'post_data', FILTER_SANITIZE_STRING, FILTER_REQUIRE_ARRAY );
+	$user_id   = bp_loggedin_user_id() ? (int) bp_loggedin_user_id() : '';
+
+	if ( ! isset( $post_data['thread_id'] ) ) {
+		$response['message'] = new WP_Error( 'bp_error_get_recepient_list_for_blocks', esc_html__( 'Missing thread id.', 'buddyboss' ) );
+		wp_send_json_error( $response );
+	}
+
+	if ( ! isset( $post_data['page_no'] ) ) {
+		$response['message'] = new WP_Error( 'bp_error_get_recepient_list_for_blocks', esc_html__( 'Invalid page number.', 'buddyboss' ) );
+		wp_send_json_error( $response );
+	}
+
+	// Get all admin ids.
+	$adminstrator_ids = function_exists( 'bb_get_all_admin_user' ) ? bb_get_all_admin_user() : '';
+
+	$moderation_type = BP_Moderation_Members::$moderation_type;
+
+	$args = array(
+		'per_page'             => bp_messages_recepients_per_page(),
+		'page'                 => (int) $post_data['page_no'],
+		'include_threads'      => array( $post_data['thread_id'] ),
+		'include_current_user' => $user_id,
+	);
+	if ( isset( $post_data['action'] ) && 'bp_load_more' === $post_data['action'] ) {
+		$args['exclude_current_user'] = $adminstrator_ids;
+	}
+
+	$results = BP_Messages_Thread::get($args);
+	if ( is_array( $results['recipients'] ) ) {
+		$count          = 1;
```

Comment:
```suggestion
		$count          = 0;
```

Maybe count should be start from `0`

Reply:
As per code reference it same 
Please check here
![image](https://user-images.githubusercontent.com/30429982/129156544-5788d913-d2be-4062-9b3c-929325f9f19f.png)